packages/blade/src/components/Alert/_decisions/decisions.md

```diff
@@ -0,0 +1,162 @@
+
+| Sub component           | Description                                                          | Required |
+| ----------------------- | -------------------------------------------------------------------- | -------- |
+| `Alert.Description`     | For setting body content                                             | ✅       |
+| `Alert.PrimaryAction`   | Renders a primary `Button`                                           |          |
+| `Alert.SecondaryAction` | Renders a button `Link`, requires `Alert.PrimaryAction` alongside \* |          |
+
+### Alert.Description
+
+| Prop       | Type          | Default     | Description | Required |
+| ---------- | ------------- | ----------- | ----------- | -------- |
+| `children` | `string`, JSX | `undefined` | Content \*  | ✅       |
+
+### Alert.PrimaryAction
+
+`Alert.PrimaryAction` renders and accepts a subset of `Button` (primary) props:
+
+| Prop       | Type       | Default     | Description                            | Required |
+| ---------- | ---------- | ----------- | -------------------------------------- | -------- |
+| `children` | `string`   | `undefined` | Content, required here unlike `Button` | ✅       |
+| `onClick`  | `function` | `undefined` | Callback                               | ✅       |
+
+### Alert.SecondaryAction
+
+`Alert.SecondaryAction` renders and accepts a subset of `Link` (button) props:
+
+| Prop       | Type          | Default     | Description | Required |
+| ---------- | ------------- | ----------- | ----------- | -------- |
+| `children` | `string`, JSX | `undefined` | Content     | ✅       |
+| `href`     | `string`      | `undefined` | Hyperlink   |          |
+| `onClick`  | `function`    | `undefined` | Callback    |          |
+
+There'll also be an internal `BaseAlert` component that handles rendering web or native version similar to other Blade components. API would be similar to `Alert`.
+
+## Usage
+
+Some example usage patterns of `Alert`.
+
+### Without title
+
+<img src="./alert-without-title.png" alt="" width="520">
+
+```jsx
+<Alert variant="notice">
+  <Alert.Description>
+    The payment was made 6 months ago, therefore you can’t issue refund to this merchant.
+  </Alert.Description>
+</Alert>
+```
+
+### Full bleed
+
+> **Note**
+>
+> Design update pending to remove all borders and border radii
+
+This full bleed layout works for all desktop, mobile and native when the `isBorderless` prop is passed
+
+<img src="./alert-mobile-full-bleed.png" alt="" width="520">
+
+```jsx
+<Alert title="International Payments Only" isBorderless>
+  <Alert.Description>
+    Use vendor payouts to quickly generate invoices.{' '}
+    <Link href="https://razorpay.com">Know More</Link>
+  </Alert.Description>
+</Alert>
+```
+
+## Open questions
+
+**Q1.** Should we use a different variant naming convention in code like `success`, `warn`, `info`, `error`? Currently this is same as design
```

Comment:
Valid argument, engineering generally should go with `success`, `warn`, `info`, `error` but in Blade, we have maintained prop consistency with design so that navigation and usability across Figma and code is seamless. Need to decide on this one as a team.

Reply:
Right, keeping the terminology same as design is the driving factor here. Will update A1 for this Q1.